api/PerformanceNavigation.json

```
@@ -67,10 +67,10 @@
               "version_added": true
             },
             "firefox": {
-              "version_added": "7"
+              "version_added": false
```

Instead of `false` here, I'd add a note like this:

```suggestion
              "version_added": "7",
              "notes": "This property can return incorrect values. See <a href='https://bugzil.la/1459711'>bug 1459711</a>."
```

What do you think?